worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/BufferStatusHub.java

```diff
@@ -73,9 +75,10 @@ protected BufferStatusNode newEmptyNode() {
   }
 
   public long avgBytesPerSec() {
-    long currentNumBytes = sum().numBytes();
+    Pair<BufferStatusNode, Integer> sumInfo = sum();
+    long currentNumBytes = sumInfo.getKey().numBytes();
     if (currentNumBytes > 0) {
-      return currentNumBytes * 1000 / (long) getCurrentTimeWindowsInMills();
+      return currentNumBytes * 1000 / (long) sumInfo.getValue();
```

Comment:
The unit of `sumInfo.getValue()` is sec now, so we don't need to times *1000 here?

Reply:
Yea, miss this